server/importPackages.ts

```diff
@@ -110,3 +110,4 @@ import '../app/reactions/server';
 import '../app/livechat/server';
 import '../app/custom/server';
 import '../app/authentication/server';
+import '../app/voip/server';
```

Comment:
```suggestion
```
Avoid creating new folders inside `app/`. We want to get rid of the package structure and have all the new code organized outside of it.

Reply:
In that case, I am not sure where to move it. Would app/api/server/v1/voip would be the right place? But in that case, this is still inside app.
Lets discuss this in the meeting.

Reply:
Agreed upon keeping it same as is after discussing. The reason is that this folder contains only APIs.

Reply:
I think I may not have explained myself properly on the meeting. I meant that creating a voip folder on `app/api/server/v1/` is fine for now, but a voip folder on `app/` is not.